hr_timesheet_activity_begin_end/models/account_analytic_line.py

```diff
@@ -31,23 +17,24 @@ class AccountAnalyticLine(models.Model):
     @api.one
     @api.constrains('time_start', 'time_stop', 'unit_amount')
     def _check_time_start_stop(self):
+        value_to_html = self.env['ir.qweb.field.float_time'].value_to_html
```

Comment:
Just a formal comment: wouldn't be more explicit take it via import?

Reply:
This way you can benefit from Odoo inheritance.